ui/selectors/assets.ts

```diff
@@ -200,3 +200,67 @@ export const getMultiChainAssets = createDeepEqualSelector(
     });
   },
 );
+
+const zeroBalanceAssetFallback = { amount: 0, unit: '' };
+
+export const getMultichainAggregatedBalance = createDeepEqualSelector(
+  (_state, selectedAccount) => selectedAccount,
+  getMultichainNetwork,
+  getMultichainBalances,
+  getAccountAssets,
+  getAssetsRates,
+  (
+    selectedAccountAddress,
+    currentNetwork,
+    multichainBalances,
+    accountAssets,
+    assetRates,
+  ) => {
+    const assetIds = accountAssets?.[selectedAccountAddress.id] || [];
+    const balances = multichainBalances?.[selectedAccountAddress.id];
+
+    let aggregatedBalance = new BigNumber(0);
+
+    assetIds.forEach((assetId: CaipAssetId) => {
+      const { chainId } = parseCaipAssetType(assetId);
+      if (chainId === currentNetwork.chainId) {
+        const balance = balances?.[assetId] || zeroBalanceAssetFallback;
+        const rate = assetRates?.[assetId]?.rate || '0';
+        const balanceInFiat = new BigNumber(balance.amount).times(rate);
+
+        aggregatedBalance = aggregatedBalance.plus(balanceInFiat);
+      }
+    });
+
+    return aggregatedBalance.toNumber();
+  },
+);
+
+export const getMultichainNativeTokenBalance = createDeepEqualSelector(
+  (_state, selectedAccount) => selectedAccount,
+  getMultichainNetwork,
+  getMultichainBalances,
+  getAccountAssets,
+  (
+    selectedAccountAddress,
+    currentNetwork,
+    multichainBalances,
+    accountAssets,
+  ) => {
+    const assetIds = accountAssets?.[selectedAccountAddress.id] || [];
+    const balances = multichainBalances?.[selectedAccountAddress.id];
+
+    let nativeTokenBalance = zeroBalanceAssetFallback;
+
+    assetIds.forEach((assetId: CaipAssetId) => {
+      const { chainId, assetNamespace } = parseCaipAssetType(assetId);
+      if (chainId === currentNetwork.chainId && assetNamespace === 'slip44') {
+        const balance = balances?.[assetId] || zeroBalanceAssetFallback;
+
+        nativeTokenBalance = balance;
+      }
+    });
```

Comment:
(Future PR) - Since we don't do any addition here, could this be replaced with a `.some` so we can short-circuit the array and not check all items?